core/mutator/entry/mutation.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Added tests for the function with an if statement. 

All other functions are simple wrappers around other tested functions.